s3po/backends/s3.py

```diff
@@ -26,7 +26,7 @@ def download(self, bucket, key, destination, retries, headers=None):
         bucket = self.get_bucket(bucket)
         # Make a file that we'll write into
         destination = CountFile(destination)
-        obj = bucket.get_key(key)
+        obj = bucket.get_key(key, validate=False)
```

Comment:
Have you confirmed that `obj` is ever `None` when `validate=False`? I believe when I've done this in the past that you get back a Key object no matter what.